web/models/terms/term.ex

```diff
@@ -31,5 +32,18 @@ defmodule CoursePlanner.Terms.Term do
     |> validate_required([:name, :start_date, :end_date, :status])
     |> cast_embed(:holidays)
     |> Statuses.update_status_timestamp(EntityStatus)
+    |> validate_changes()
   end
+
+  defp validate_changes(%{data: %{start_date: start_date, end_date: end_date},
+      changes: changes, errors: []} = changeset) do
+    {:ok, st} = Date.cast(Map.get(changes, :start_date) || start_date)
```

Comment:
If you wanna grab the value regardless of having changes you may use `Ecto.Chanfeset.get_field` instead